### PR TITLE
Update zh.js  (v3.8.1)

### PR DESCRIPTION
**Pull request by Awesome-Boy-001**
### Major updates/fixes on Chinese, Simplified version of translation, such as pushed in professionalized words & phrases, added in new sentences & words, and fixed some confusing parts.

_Note: Please view the latest commit since the first was just a test._

**Contact:**
Discord ID: Awesome_Boy#7555

### CODE REVIEW

**Comment by mty22** [comment]
Perfect, thank you.

I'll find a native speaker to review your changes and get back to you on this pull request.

File: zh.js [hunk 1]
```diff
@@ -1,747 +1,747 @@
...
-server.config.noSecondary 禁用服务器 
-server.config.noStreaks 禁用连杀 
-server.config.canSlide Sliding 
-server.config.wallJP 墙壁跳跃倍率 
-server.config.autoJump 自动跳跃 
-server.config.bDrop 子弹下坠 
-server.config.thirdPerson 第三人称 
-server.config.requireCaptcha 需要Captcha验证 
-server.config.nameTags 隐藏名牌 
+module.exports = `
```
Comment (mty22) [inline zh.js:1]: need to remove white space here

File: zh.js [hunk 2]
```diff
@@ -1,747 +1,747 @@
...
+generic.leave 退出 
+generic.default 默认 
+generic.submit 提交 
+generic.create 创建 
+generic.random 随机 
+generic.failed 失败 
+generic.success 成功! 
+generic.name 姓名 
+generic.level 等级 
+generic.score 得分
```
Comment (mty22) [inline zh.js:32]: need to remove white space here

File: zh.js [hunk 3]
```diff
@@ -1,747 +1,747 @@
...
+settings.controls.press 按下任意按键 
+settings.local.header 本地化 <span id="requiresRestart"><span style="color: #eb5656">*</span> 需要重启游戏 </span> 
+settings.local.region 默认区域 
+settings.local.lang 语言 
+settings.network.header 网路 
+settings.network.sendRate 高服务器应答率 
+settings.network.netRerouting 自动路由 
+settings.network.lagComp 延迟修正 (Lag Compensation) 
+settings.popup.header 弹窗 
+settings.popup.freeKRPop 显示免费KR弹窗
```
Comment (mty22) [inline zh.js:95]: need to remove white space here

File: zh.js [hunk 4]
```diff
@@ -1,747 +1,747 @@
...
+settings.popup.header 弹窗 
+settings.popup.freeKRPop 显示免费KR弹窗 
+settings.experimental.header 实验性功能 
+settings.experimental.aimFreezeFix 瞄射冻结修复 
+settings.experimental.mouseFlickFix 鼠标抽动修复 
+settings.experimental.shaderRendering 实例渲染
+settings.dev.header 开发者 
+settings.dev.forceUpdate 强制更新皮肤 
+settings.dev.forceRegen 永再生玩家 
+settings.dev.extraBan 额外禁令信息
```
Comment (mty22) [inline zh.js:103]: need to remove white space on these lines

File: zh.js [hunk 5]
```diff
@@ -1,747 +1,747 @@
...
+settings.local.lang 语言 
+settings.network.header 网路 
+settings.network.sendRate 高服务器应答率 
+settings.network.netRerouting 自动路由 
+settings.network.lagComp 延迟修正 (Lag Compensation) 
+settings.popup.header 弹窗 
+settings.popup.freeKRPop 显示免费KR弹窗 
+settings.experimental.header 实验性功能 
+settings.experimental.aimFreezeFix 瞄射冻结修复 
+settings.experimental.mouseFlickFix 鼠标抽动修复
```
Comment (mty22) [inline zh.js:98]: need to remove white space here

File: zh.js [hunk 6]
```diff
@@ -1,747 +1,747 @@
...
+settings.editing.menuSaturation 颜色饱和度 (用户界面) 
+settings.editing.hue 色调 (游戏中) 
+settings.editing.menuHue 色调 (用户界面) 
+settings.editing.vignette 晕影效果（使屏幕边缘颜色变暗) 
+settings.editing.feedLimit 击杀列表长度限制 
+settings.editing.depth 深度图模式 
+settings.editing.green 绿幕模式 
+settings.editing.tracercol 弹道轨迹 
+settings.editing.tracer 弹道轨迹 
+settings.editing.tracer.offset 示踪轨迹
```
Comment (mty22) [inline zh.js:346]: need to remove white space here

File: zh.js [hunk 7]
```diff
@@ -1,747 +1,747 @@
...
+settings.editing.tracercol 弹道轨迹 
+settings.editing.tracer 弹道轨迹 
+settings.editing.tracer.offset 示踪轨迹 
+settings.shaders.header 着色器输入 
+settings.mods.header 自定义 
+settings.mods.load 允许加载插件 
+settings.mods.logo 允许变更游戏商标 
+settings.mods.auto 自动加载插件 
+settings.mods.borders 瞄准镜边界 
+settings.mods.borderCol 瞄准镜边界
```
Comment (mty22) [inline zh.js:353]: need to remove white space here

File: zh.js [hunk 8]
```diff
@@ -1,747 +1,747 @@
...
+social.market.price.high 最高价格 
+social.trading.submitting 提交报价中... 
+social.market.gift.amount 赠送金额 
+social.market.fee.gift 无费用 
+social.market.gift 赠送KR 
+social.market.gift.msg 留言 
+respawn.auto 自动重生在 {0} 秒内
+custom.host.ranked 排位赛中无法举办新游戏。
+funds.insufficient 资金不充足 
+funds.out 您目前已用完您的 Krunkies.
```
Comment (mty22) [inline zh.js:700]: need to remove white space on these lines

**Review by mty22 — CHANGES_REQUESTED** [review]
Hey there, just need to remove the whitespace on the end of these lines.

Can you fix and push another update please?

**Comment by mty22** [comment]
@Awesome-Boy-001 need you to check my comment in https://discord.com/channels/448194623580667916/831383180086607922/833722205635739690

this PR is on hold until that is checked.

**Comment by WpyGG4125** [comment]
Completed file translation using both files, but idk what to do next

**Comment by mty22** [comment]
there's two active PRs: https://github.com/mty22/krunker.io-translations/pull/104/  + this one

**Comment by mty22** [comment]
I've made comments in Discord, please have a look:

https://discord.com/channels/448194623580667916/831383180086607922/834227613236461588
https://discord.com/channels/448194623580667916/831383180086607922/834409630837768222

**Comment by mty22** [comment]
Hi @Awesome-Boy-001 

I had a chat with ReHou (original Simplified Chinese translator), and asked him to check both pull requests since it became apparent that neither teams would work together.

This is his feedback:

image

Thank you for the work, although for the sake of the game and the trusted feedback that I've received; I believe in this scenario that it's better we roll with https://github.com/mty22/krunker.io-translations/pull/104

**Pull request closed without merging**